lib/evmone/baseline.cpp

```diff
@@ -276,7 +323,14 @@ evmc_result execute(const VM& vm, ExecutionState& state, const CodeAnalysis& ana
     if (INTX_UNLIKELY(tracer != nullptr))
         dispatch<true>(cost_table, state, tracer);
     else
-        dispatch<false>(cost_table, state);
+    {
+#if EVMONE_CGOTO_SUPPORTED
+        if (vm.cgoto)
```

Comment:
I wonder if making this `else if (INTX_LIKELY(vm.cgoto))` can trigger any better layout? Since the default case is cgoto unless disabled.

Reply:
We don't want to use likely/unlikely here, because there is a chance compiler will compile the unlikely branch for code size. I want them to have equal chances because they are competing against each other in benchmarks.